bridge-fungible-token/Cargo.toml

```diff
@@ -9,7 +9,8 @@ license = "Apache-2.0"
 fuel-core = "0.10.1"
 fuel-core-interfaces = "0.10.1"
 fuel-crypto = "0.5"
```

Comment:
`fuel-crypto` may also no longer be necessary as I don't think we manually `hash` any values anymore

Reply:
We still use it here:
```
pub async fn get_contract_message_script() -> (Vec<u8>, Bytes32) {
    let script_bytecode = std::fs::read(CONTRACT_MESSAGE_SCRIPT_BINARY).unwrap();
    let script_hash = Hasher::hash(&script_bytecode.clone());
    (script_bytecode, script_hash)
}
```

Reply:
The one place that the function is called, the hash isn't used